client.lua

```diff
@@ -132,21 +143,23 @@ end, 1000)
 
 local function StartFueling(vehicle)
     isFueling = true
+    local ped = PlayerPedId()
     local price = 0
     local moneyAmount = exports.ox_inventory:Search(2, 'money')
 
     local Vehicle = Entity(vehicle).state
     local fuel = Vehicle.fuel
 
     if 100 - fuel < ox.refillValue then
+        isFueling = false
```

Comment:
This already gets set to false by the progress bar callback function, why set it again?

Reply:
@LukeWasTakenn Because the progress bar is not started at that point and if it returns the notify, it will never trigger the progress bar and the callback. Am I wrong?

Reply:
the progress bar callback function will always run after the progress bar is done either by cancelling or letting it reach the end

Reply:
But the progress bar will never run if the function returns on notify, check next line